bench.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
```

Comment:
Can you add this bench to `scripts` in package.json as `"bench": "node bench.js"`?